### DISCUSSION TITLE
How to format text (mix several styles in one text object)?

### DISCUSSION THREAD

**Discussion by ghost** [slint-ui/slint#3756] [Q&A] [answered]
I want something like **bold** or *italic* per one word in sentence (for emphasis purposes) but doesn't see how to do it. I know about the properties of [Text element](https://slint.dev/releases/1.2.2/docs/slint/src/language/builtins/elements#text) but it they give opportunity to change the style of the whole text element... Is it even possible to format *part* of the text? Or I should implement it by myself with several text objects...?

**Comment by bjorn** [comment, marked as answer]
That feature is probably covered by #2723 (Rich text editor). Currently I think you have to use multiple Text objects.
